app/facades/movie_facade.rb

```diff
@@ -0,0 +1,15 @@
+class MovieFacade
+  def self.top_forty
+    json = MovieService.top_forty
+    @movies = json.map do |movie_data|
```

Comment:
I even wonder if line 4 and line 11 map block can be pulled out into a single method?

Reply:
I think that's a brilliant idea!  Let's do that!